### ISSUE TITLE
maim support gem and is maimed

### ISSUE THREAD

**Issue by Phil-Morley** [Openarl/PathOfBuilding#598]
Hi, thankyou so much for a great tool!

The new maim support gem has increased damage against maimed enemies but choosing is maimed in the configuration doesnt increase the dps.

**Comment by dein0s** [comment]
It increases only physical damage taken by enemies and it's working as intended in v1.4.61.
Re-check you configuration and main skill setup.

**Comment by Phil-Morley** [comment]
thanks, running it with frostblades 100% cold so there is no buff. sorry about that

**Comment by Phil-Morley** [comment]
sorry just rethinking and trying to understand this, I assumed it was because 100% cold (because some things like leech physical attack damage dont work) but 
_Supported Skills deal (15-30)% more Physical Damage_
this works for the cold damage and gives a nice boost
_Enemies Maimed by Supported Skills take (10-14)% increased Physical Damage_
this one is the same language physical damage rather than physical attack damage but it doesnt work for cold?

**Comment by dein0s** [comment]
> Supported Skills deal (15-30)% more Physical Damage
this works for the cold damage and gives a nice boost

TL;DR - it doesn't work for the cold damage, it still buffs only phys damage. You can read more detailed explanation on how damage conversion works on wiki: https://pathofexile.gamepedia.com/Damage_conversion

> Enemies Maimed by Supported Skills take (10-14)% increased Physical Damage
this one is the same language physical damage rather than physical attack damage but it doesnt work for cold?

It's not same, wording means a lot in this game. After all conversions are done (read the article from above), in your case you are doing 100% pure cold damage (0% physical) and therefore `Enemies Maimed by Supported Skills take (10-14)% increased Physical Damage` does not affect your dps at all as enemies takes only cold damage.

**Comment by Phil-Morley** [comment]
ok thanks I'm just surprised that the 'more physical damage' works but 'increased physical damage' doesnt
